client/TracyProfiler.cpp

```diff
@@ -440,25 +443,29 @@ static const char* GetHostInfo()
     static char buf[1024];
     auto ptr = buf;
 #if defined _WIN32
-    t_RtlGetVersion RtlGetVersion = (t_RtlGetVersion)GetProcAddress( GetModuleHandleA( "ntdll.dll" ), "RtlGetVersion" );
-    if( !RtlGetVersion )
+#  ifdef TRACY_UWP
+    auto GetVersion = &::GetVersionEx;
+#  else
+    auto GetVersion = (t_RtlGetVersion)GetProcAddress( GetModuleHandleA( "ntdll.dll" ), "RtlGetVersion" );
+#  endif
+    if( !GetVersion )
     {
-#  ifdef __MINGW32__
+#    ifdef __MINGW32__
```

Comment:
Is this (and following) indent change needed?

Reply:
No, that was just left there by mistake in https://github.com/wolfpld/tracy/pull/324/commits/a0a56eaf9eafc997cd3ba41ff121bd9f8a1779fc

Fixed: https://github.com/wolfpld/tracy/pull/324/commits/aa44cedcd2d539365982df6c8fb7cd818fae51b5